CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.0.7 [#58](https://github.com/patterninc/muffin_man/pull/58)
```

Comment:
I think we can go with minor version update instead of patch version update as its a new non-breaking functionality that we are adding.

Refer: Summary section from here - https://semver.org/
```
Given a version number MAJOR.MINOR.PATCH, increment the:

MAJOR version when you make incompatible API changes
MINOR version when you add functionality in a backward compatible manner
PATCH version when you make backward compatible bug fixes
```